charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Infrastructure Agent as a DaemonSet
 name: newrelic-infrastructure
-version: 1.6.5
```

Comment:
Do we need to bump the chart version here? If we're not pulling new deps or breaking the format of the values I'd leave it as it is.

Reply:
NVM, seems that version is the importat one to bump.

Reply:
Discussed offline